io/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not 100% sure: we can imagine a developer will use hdfs io (written in java) with the scala sdk for instance.
It's what we do in spark: spark-streaming connectors can be written in java or scala, and use in spark-streaming java or scala: both are not really coupled.